setup.py

```diff
@@ -70,7 +70,7 @@
     'recommonmark',
     'sphinx',
     'sphinx_rtd_theme',
-    'yapf',
+    'yapf==0.25.0',
```

Comment:
@ryanjulian If we are setting yapf to 0.28.0 in pre-commit, this should also be 0.28.0, right?

Reply:
yes -- we should strive to have them all match.